vm/src/obj/objdict.rs

```diff
@@ -471,17 +471,15 @@ impl PyDictRef {
     }
 }
 
-impl ItemProtocol for PyDictRef {
-    fn get_item<T: IntoPyObject + DictKey + Copy>(&self, key: T, vm: &VirtualMachine) -> PyResult {
+impl<T> ItemProtocol<T> for PyDictRef
+where
+    T: IntoPyObject + DictKey,
```

Comment:
```suggestion
    T: DictKey,
```
`DictKey: IntoPyObject` anyway